gradle/artifact-settings.gradle

```
@@ -10,7 +10,13 @@ ext {
     mapboxArtifactScmUrl = 'scm:git@github.com:mapbox/mapbox-navigation-android.git'
     mapboxArtifactLicenseName = 'BSD'
     mapboxArtifactLicenseUrl = 'https://opensource.org/licenses/BSD-2-Clause'
-    versionName = project.property('POM_VERSION_NAME')
+    releaseSdk = project.property('RELEASE_SDK')
+    if ( releaseSdk == 'core' ) {
+        versionName = System.getenv('POM_CORE_VERSION_NAME') ?: '1.0.0-SNAPSHOT'
```

```suggestion
        versionName = System.getenv('POM_CORE_VERSION_NAME') ?: project.property('POM_CORE_SNAPSHOT_VERSION_NAME')
```

and add `POM_CORE_SNAPSHOT_VERSION_NAME` into the root `gradle.properties` file.

---

Could we also add third condition? Something similar to https://github.com/mapbox/mapbox-navigation-android/blob/3104a0118c40d0780969b2dce13c5ac48e05d708/gradle/artifact-settings.gradle#L17 I had to add that to publish locally 👀 

```diff
diff --git a/build.gradle b/build.gradle
index a1a38d07a..7321f6529 100644
--- a/build.gradle
+++ b/build.gradle
@@ -5,6 +5,7 @@ buildscript {
   apply from: "${rootDir}/gradle/dependencies.gradle"
 
   repositories {
+    mavenLocal()
     google()
     mavenCentral()
     jcenter()
@@ -37,6 +38,7 @@ task testReport(type: TestReport, group: 'Build') {
 
 allprojects {
   repositories {
+    mavenLocal()
     google()
     mavenCentral()
     jcenter()
diff --git a/gradle.properties b/gradle.properties
index 81c91b2b9..7742c06c4 100644
--- a/gradle.properties
+++ b/gradle.properties
@@ -1,6 +1,8 @@
 # Project-wide Gradle settings.
 VERSION_NAME=0.43.0-SNAPSHOT
 GROUP=com.mapbox.mapboxsdk
+POM_CORE_VERSION_NAME=1.0.0-qa.5
+POM_UI_VERSION_NAME=1.0.0-qa.2
 
 POM_URL=https://github.com/mapbox/mapbox-navigation-android
 POM_SCM_URL=https://github.com/mapbox/mapbox-navigation-android
diff --git a/gradle/artifact-settings.gradle b/gradle/artifact-settings.gradle
index 3783c0b4e..e136a93ff 100644
--- a/gradle/artifact-settings.gradle
+++ b/gradle/artifact-settings.gradle
@@ -12,10 +12,10 @@ ext {
     mapboxArtifactLicenseUrl = 'https://opensource.org/licenses/BSD-2-Clause'
     releaseSdk = project.property('RELEASE_SDK')
     if ( releaseSdk == 'core' ) {
-        versionName = System.getenv('POM_CORE_VERSION_NAME') ?: '1.0.0-SNAPSHOT'
+        versionName = project.property('POM_CORE_VERSION_NAME') ?: '1.0.0-SNAPSHOT'
     }
     else {
-        versionName = System.getenv('POM_UI_VERSION_NAME') ?: '1.0.0-SNAPSHOT'
+        versionName = project.property('POM_UI_VERSION_NAME') ?: '1.0.0-SNAPSHOT'
     }
 
     mapboxBintrayUserOrg = 'mapbox'
```